benchmarking/hist_filling.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="us-ascii"?>
+
+<marlin>
+  <execute>
+    <processor name="CPUCrunch"/>
+    <if condition="!${RunNative}">
+      <processor name="HistFilling"/>
+    </if>
+    <if condition="${RunNative}">
+      <processor name="HistFillingNative"/>
+    </if>
+  </execute>
+  <constants>
+    <constant name="TriggerUnpacking" value="false"/>
+    <constant name="RunNative" value="false"/>
+  </constants>
+
+  <global>
+    <parameter name="Verbosity"> MESSAGE </parameter>
+    <parameter name="ColoredConsole"> 0 </parameter>
+    <parameter name="Concurrency">3</parameter>
+  </global>
+
+  <bookstore>
+    <parameter name="OutputFile">HistFill.root</parameter>
+    <parameter name="DefaultMemoryLayout">Copy</parameter>
+  </bookstore>
+
+  <datasource type="LCIO">
+    <parameter name="LCIOInputFiles">  </parameter>
+    <parameter name="MaxRecordNumber" value="0"/>
+    <parameter name="SkipNEvents" value="0"/>
+    <parameter name="LazyUnpack" value="true"/>
+  </datasource>
+
+  
+  <geometry type="EmptyGeometry" />
+
+  <processor name="UnpackEvent" type="LCIOEventUnpacking"/>
```

Comment:
You create the processor but you don't have it in the execute section. Do you want to add it in the execute section or do you want to remove totally from the config ?